zebrad/src/commands/start.rs

```diff
@@ -126,6 +137,10 @@ impl StartCmd {
             mempool_crawl_result = mempool_crawler_task_handle.fuse() => mempool_crawl_result
                 .expect("unexpected panic in the mempool crawler")
                 .map_err(|e| eyre!(e)),
+
+            tx_gossip_result = tx_gossip_task_handle.fuse() =>tx_gossip_result
```

Comment:
Typo: rustfmt doesn't work inside macros
```suggestion
            tx_gossip_result = tx_gossip_task_handle.fuse() => tx_gossip_result
```